src/ChangeStream.php

```diff
@@ -143,8 +143,8 @@ private function extractResumeToken($document)
         if ($document === null) {
             throw new ResumeTokenException("Cannot extract a resumeToken from an empty document");
         }
-        if ($document instanceof Serializable) {
-            return $this->extractResumeToken($document->bsonSerialize());
+        if ($document instanceof \Serializable) {
```

Comment:
I believe this should have been left as-is, while adding `use MongoDB\BSON\Serializable;` above.